contents/julia_basics.md

```diff
@@ -1037,196 +1051,175 @@ We also have some **syntax aliases** for the most common elements in array const
   Note that the default type is `Float64` which can be changed if necessary:
 
     ```jl
-    sco(
-    """
-    my_vector_zeros = zeros(10)
-    """
-    )
+    s = """
+        my_vector_zeros = zeros(10)
+        """
+    sco(s)
```

Comment:
```suggestion
    sco("my_vector_zeros = zeros(10)")
```
Would have been better, but I forgot to do it here. Maybe next time. Later in the PR, I did go for the shorter form.

Reply:
There are several like this in the file. Doesn't make sense to change only one...